modules/odc-interacting-with-applications-and-components.adoc

```diff
@@ -18,3 +18,13 @@ This feature is available only when you create applications using the *From Git*
 * Hover your cursor over the lower left icon on the Pod to see the name of the latest build and its status. The status of the application build is indicated as *New* (image:odc_build_new.png[title="New Build"]), *Pending* (image:odc_build_pending.png[title="Pending Build"]), *Running* (image:odc_build_running.png[title="Running Build"]), *Completed* (image:odc_build_completed.png[title="Completed Build"]), *Failed* (image:odc_build_failed.png[title="Failed Build"]), and *Canceled* (image:odc_build_canceled.png[title="Canceled Build"]).
 * Use the *Shortcuts* menu listed on the upper-right of the screen to navigate components in the *Topology* view.
 * Use the *List View* icon to see a list of all your applications and use the *Topology View* icon to switch back to the *Topology* view.
+* Use the *Display Options* drop-down menu to configure the Topology view of the various application groupings. The options are available depending on your project structure and installed Operators:
+** *Show*:
+*** *Pod Count*: Select to show the number of pods of a component in an component icon.
+*** *Labels*: Toggle to show or hide component labels.
```

Comment:
We can also add the below two entries under show option
```
Event sources - Toggle to show or hide the event sources.
Virtual Machines - Toggle to show or hide the virtual machines.
```

Reply:
yep, this note is basically in the leading sentence, so ill leave it out. ty :)